src/index.html

```diff
@@ -6,9 +6,91 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <div class="shell">
```

Comment:
shell it's a strange name, usually we name such div's as `container` or `wrapper` but ok
what is not ok - it's more logic if header container will be outside of header, not inside 🙂

Reply:
I tried to wrap header in a container. But then the tests didn't work(